.github/workflows/built-release.yml

```diff
@@ -0,0 +1,14 @@
+name: Built Release
```

Comment:
Since you are removing the built-tag, the https://github.com/alleyinteractive/create-wordpress-plugin#create-wordpress-plugin should be updated to reflect this change.

Reply:
Added https://github.com/alleyinteractive/create-wordpress-plugin/tree/built-release?tab=readme-ov-file#releasing-the-plugin !